yjit/src/codegen.rs

```diff
@@ -815,6 +826,8 @@ pub fn gen_single_block(blockref: &BlockRef, ec: EcPtr, cb: &mut CodeBlock, ocb:
         // If we can't compile this instruction
         // exit to the interpreter and stop compiling
         if status == CantCompile {
+            let mut block = jit.block.borrow_mut();
+
```

Comment:
Looks good so far apart from this minor comment

Reply:
Because only one mutable borrow could be live and we have other places where we need to mutably borrow. We don't pass the one `RcMut` around so calls to `borrow_mut` deeper in the call graph was panicking. It's dynamically borrow checking. This is the downside of using RefCell unfortunately. If it gets out of hand we could refactor to use statically checked borrows.